translate/__version__.py

```diff
@@ -19,16 +19,16 @@
 
 """This file contains the version of the Translate Toolkit."""
 
-build = 14001
+build = 20000
 """The build number is used by external users of the Translate Toolkit to
 trigger refreshes.  Thus increase the build number whenever changes are made to
 code touching stats or quality checks.  An increased build number will force a
 toolkit user, like Pootle, to regenerate it's stored stats and check
 results."""
 
-sver = "1.14.0-rc1"
+sver = "2.0.0-beta1"
```

Comment:
i wonder if we should rather follow semantic versioning where this would be 2.0.0b1 IIRC